### PR TITLE
Added Dedicated reader role

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://trello.com/c/0pGDMQh4/1143-3-dedicated-provide-a-read-only-version-of-dedicated-admin?menu=filter&filter=label:documentation

@wgordon @abhgupta PTAL

### CODE REVIEW

**Comment by wgordon17** [comment]
Hey @ahardin-rh, sorry...but this isn't quite accurate. The dedicated-reader is the dedicated-admin counterpart, not a version of cluster-reader. So the page you currently edited can probably be reverted. You could maybe add a section [here](https://docs.openshift.com/dedicated/admin_guide/index.html) about Dedicated Reader Role. And also [here](https://docs.openshift.com/dedicated/admin_guide/service_accounts.html#dedicated-admin-granting-service-accounts-access) for a `dedicated-reader` project. And then [here](https://docs.openshift.com/dedicated/getting_started/dedicated_administrators.html#gs-dedicated-admin-managing-admin-list) for adding users to the `dedicated-readers` group...I think that should be it! 👍

**Comment by ahardin-rh** [comment]
@wgordon17 Updated. PTAL. Thanks!

**Comment by ahardin-rh** [comment]
@wgordon17 @yasun1 Updated PTAL.

File: admin_guide/service_accounts.adoc [hunk 1]
```diff
@@ -108,6 +108,9 @@ Users that are members of the *dedicated-admins* group, and thus have been
 granted the *dedicated-admin* role, have `edit` access to the *dedicated-admin*
 project. This allows these users to manage the service accounts in this project
 and create new ones as needed.
+
+Users with a *dedicated-reader* role are granted edit and view access to the
+*dedicated-readers* project and view-only access to the other projects.
```
Comment (yasun1) [inline admin_guide/service_accounts.adoc]: There is a tiny point need to be updated. The project is `dedicated-reader`, but not dedicated-reader**s**.

File: getting_started/dedicated_administrators.adoc [hunk 2]
```diff
@@ -140,6 +140,31 @@ If this group is currently empty or if you need assistance editing group
 membership, open a support case on the
 link:https://access.redhat.com/support/[Red Hat Customer Portal].
 
+[[managing-dedicated-readers-group]]
+== Managing the Dedicated Readers Group
+
+Users in the *dedicated-readers* group are granted view-only access to
+*dedicated-readers* projects.
```
Comment (yasun1) [inline getting_started/dedicated_administrators.adoc]: This sentence should be synced with above.

File: getting_started/dedicated_administrators.adoc [hunk 3]
```diff
@@ -140,6 +140,31 @@ If this group is currently empty or if you need assistance editing group
...
+
+To view a list of current dedicated readers by user name, you can use the
+following command:
```
Comment (yasun1) [inline getting_started/dedicated_administrators.adoc]: The below two `oc adm` commands only can be run successfully for the user who has `dedicated-admin` or higher authority. I am not sure whether needs add some words at here, it is up to your decision.

**Comment by ahardin-rh** [comment]
@wgordon17 PTAL

**Comment by ahardin-rh** [comment]
@wgordon17 Can you please review? Thanks!

**Review by wgordon17 — APPROVED** [review]
Sorry for the delay, LGTM! 👍

**Comment by ahardin-rh** [comment]
Thanks! :fireworks:

**Pull request merged**